packages/db/tests/translate/test-executions.js

```diff
@@ -2,7 +2,79 @@ const test = require('ava');
 const cryptoRandomString = require('crypto-random-string');
 
 const { removeNilProperties } = require('@cumulus/common/util');
-const { translateApiExecutionToPostgresExecution } = require('../../dist/translate/executions');
+const {
+  translateApiExecutionToPostgresExecution,
+  translatePostgresExecutionToApiExecution,
+} = require('../../dist/translate/executions');
+
+test.only('translatePostgresExecutionToApiExecution translates a Postgres execution to an API record', async (t) => {
```

Comment:
Reminder to remove `only` 😄

Reply:
Ugh.  I had to remove the commit hook because we *have* tests that deliberately *are* .only :|

Reply:
edd75a016